airflow/providers/amazon/aws/utils/tags.py

```diff
@@ -24,7 +24,7 @@ def format_tags(source: Any, *, key_label: str = "Key", value_label: str = "Valu
     If given a dictionary, formats it as an array of objects with a key and a value field to be passed to boto
     calls that expect this format.
     Else, assumes that it's already in the right format and returns it as is. We do not validate
-    the format here since it's done by boto anyway, and the error wouldn't be clearer if thrown from here.
```

Comment:
"wouldn't" seems perfectly reasonable here and is spelled correctly, so this is strange. However, "would not" works just as well, and if it makes the tests more stable then I guess why not :shrug:

Reply:
I tried add into `docs/spelling_wordlist.txt` different combination: `wouldn`, `wouldn'`, `wouldn't` but without any luck 😒

Reply:
That **wouldn't** (ehm!) work ... :D. I think the spelling wordlist acts on "words" only and ' is not considered as a letter.

Reply:
Yeah, that is the reason why I replace as most easier option.

Need to try to found again why this kind of errors not detected in CI, for example this one also wasn't detected in CI, https://github.com/apache/airflow/pull/28587

Reply:
It's (ehm! :laughing: ) also curious that it accepts the concatenated word `it's` in the same sentence but it doesn't like `wouldn't` :thinking:

Reply:
When i tried to found the nature of this spellcheck errors I found this comment funny: https://github.com/sphinx-contrib/spelling/issues/126#issuecomment-1155761023